### ISSUE TITLE
Adding a new feature for reward sketching 

### ISSUE THREAD

**Issue by ankurhanda** [heartexlabs/label-studio#814]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I'd like to have a way to create reward sketches like done here https://sites.google.com/view/data-driven-robotics/ on video data.

**Describe alternatives you've considered**
I have not seen anything close to this in the repo.

**Additional context**
These reward sketches are useful to have to train RL agents. This is the closest you can get to have a supervised signal for robotics.

**Comment by makseq** [comment]
@ankurhanda What kind of reward sketches do you want to have? 
LS has ratings, checkboxes and text areas, so you can have [something like this](https://labelstud.io/playground/?config=%3CView%3E%3Cbr%3E%20%20%3CHyperText%20name%3D%22video%22%20value%3D%22%24video%22%2F%3E%3Cbr%3E%20%20%3Cbr%3E%20%20%3CHeader%20value%3D%22Rating%22%2F%3E%3Cbr%3E%20%20%3CRating%20name%3D%22rating%22%20toName%3D%22video%22%2F%3E%3Cbr%3E%20%20%3Cbr%3E%20%20%3CHeader%20value%3D%22Choice%22%2F%3E%3Cbr%3E%20%20%3CChoices%20name%3D%22type%22%20toName%3D%22video%22%20choice%3D%22single-radio%22%3E%3Cbr%3E%20%20%20%20%3CChoice%20value%3D%22Bad%20try%22%20%2F%3E%3Cbr%3E%20%20%20%20%3CChoice%20value%3D%22Good%20try%22%20%2F%3E%3Cbr%3E%20%20%3C%2FChoices%3E%3Cbr%3E%20%20%3Cbr%3E%20%20%3CHeader%20value%3D%22Describe%22%2F%3E%3Cbr%3E%20%20%3CTextArea%20name%3D%22text%22%20toName%3D%22video%22%20%2F%3E%3Cbr%3E%20%20%3Cbr%3E%3C%2FView%3E) for video.

**Comment by ankurhanda** [comment]
@makseq I'd like to draw a continuous function with X-axis being the frame number and Y-axis being the reward. To clarify what that means check the reward sketching section in https://sites.google.com/view/data-driven-robotics/ where you slide your mouse along and it draws the curves for you.

**Comment by makseq** [comment]
Thanks! Now I've got it. Very cool feature!

**Comment by ankurhanda** [comment]
Do you think it is worth adding that in label-studio? 

I implemented it in python already (using matplotlib) but it's fairly simple and often requires more clicking.

**Comment by ankurhanda** [comment]
some of this is inspired from https://github.com/yuma-m/matplotlib-draggable-plot/blob/master/draggable_plot.py

**Comment by makseq** [comment]
Thank you very much! But we need to re-implement it in JS React for LS.

**Comment by ankurhanda** [comment]
Yes, sure. Let me know if you ever get to it. I'm happy to do testing for you. Thanks a lot for listening.

**Comment by makseq** [comment]
https://labelstud.io/tags/number.html#main - we introduced Number tag, it can help here.
